src/SourceLocator/Ast/FindReflectionsInTree.php

```diff
@@ -87,7 +87,7 @@ public function __construct(
                 $this->astConversionStrategy = $astConversionStrategy;
             }
 
-            public function enterNode(Node $node) : void
```

Comment:
These should be kept

Reply:
This method inherits from a method that has a non-`void` return type

Reply:
I've added an `@inheritDoc` annotation instead

Reply:
Btw this issue is documented here: https://github.com/phpstan/phpstan/issues/729#issuecomment-361737775

Reply:
Yeah, ok - makes sense to drop these at least for now :+1:

Reply:
Will revert this change though, per https://github.com/Roave/BetterReflection/pull/394#issuecomment-361956061